facdb/datasets.yml

```diff
@@ -94,7 +94,6 @@ datasets:
       - nycourts_courts.sql
 
   - name: nysdec_lands
-    version: 20210415
```

Comment:
can you restore this to the `datasets.yml` from the `2022-Q2-Build` so that all changes to datasets.yml just come from the PR from the 534 branch?

Reply:
like restore the whole file with git checkout